e2e/fixtures/events/claimantResponseSpecCui.js

```diff
@@ -0,0 +1,132 @@
+const {listElement} = require('../../api/dataHelper');
```

Comment:
Please add your tests in this file for the new fields you've created

Reply:
done.